osquery/utils/darwin/plist.mm

```diff
@@ -277,4 +283,22 @@ Status pathFromPlistAliasData(const std::string& data, std::string& result) {
   CFRelease(bookmark);
   return status;
 }
+
+/// Parse a nested (base-64 encoded) plist's alias data for its path
+/// Observed experimentally with /Library/Preferences/com.apple.alf.plist
+Status pathFromNestedPlistAliasData(const std::string& data,
```

Comment:
Do you mind uploading test data into `./tests` and making a unit test for this new parsing. 

I can +1 to the observation, it's the same format on a 10.14.6 test device.

Reply:
Yeah, let me see if I can whip up a sanitized version of a local sample.